models/models_methods.go

```diff
@@ -777,28 +779,17 @@ func validateDAGPipeline(pipelineID string, pipeline *PipelineModel, config *Bit
 			return fmt.Errorf("workflow (%s) defined in pipeline (%s) is a utility workflow", pipelineWorkflowID, pipelineID)
 		}
 
-		isParallelizedWorkflow := pipelineWorkflow.Parallel != nil
+		isParallelizedWorkflow := pipelineWorkflow.Parallel != ""
 		if isParallelizedWorkflow {
-			if *pipelineWorkflow.Parallel < 1 {
```

Comment:
These validations remained the same (for integer parallel value), just refactored the validations to a dedicated func.